cedar-policy-core/src/extensions/ipaddr.rs

```diff
@@ -303,13 +314,10 @@ fn str_contains_colons_and_dots(s: &str) -> Result<(), String> {
 /// Cedar string
 fn ip_from_str(arg: Value) -> evaluator::Result<ExtensionOutputValue> {
     let str = arg.get_as_string()?;
-    let function_name = names::IP_FROM_STR_NAME.clone();
     let arg_source_loc = arg.source_loc().cloned();
-    let ipaddr = ExtensionValueWithArgs::new(
-        Arc::new(IPAddr::from_str(str.as_str()).map_err(extension_err)?),
-        function_name,
-        vec![arg.into()],
-    );
+    let ipaddr = RepresentableExtensionValue::new(Arc::new(
+        IPAddr::from_str(str.as_str()).map_err(extension_err)?,
```

Comment:
Is it possible to get `str` in a different way so we don't have to `as_str()` here? IIUC we need it because we do `arg.get_as_string()` before.